translations/Hindi/special-terms.md

```diff
@@ -0,0 +1,160 @@
+which mean mute, but in the context as a person who cannot speak. , इस 
+```stream``` को म्यूट / अनम्यूट करें (Mute / 
+unmute this ```stream```)
+* **Deactivate/Reactivate** : निष्क्रिय 
+करें / पुन: सक्रिय करें , These two words are 
+direct translations and there is no alternative translation to either of them 
+which we can use in Zulip as they are long phrases , अपने खाते 
+को निष्क्रिय / पुनः सक्रिय 
+करें (Deactivate / reactivate your account)
+* **Search** : खोज , 
+खोज is used in many search engines as well, and is a well known word. 
+Other translations like तलाश and जाँच are used in the context of 
+security, where people are searched, therefore cannot be used here. , 
+आपने ```search item``` के लिए खोजा (You searched for 
+```search item```)
+* **Pin** : पिन , Transliteration used here as well for two 
+reasons: Translations of pin mean all different kinds of needles and the like, 
+and to convert to verb form we can just add करो. , इस पोस्ट 
+को पिन करें (Pin this post)
+* **Mention/@mention** : 
+उल्लेख / @ उल्लेख , उल्लेख is used as it is 
+the common translation of the word and sounds formal. Another translation, 
+कथन is more often associated with quoting from a story. , 
+उपयोगकर्ता का उल्लेख करें (Mention 
+user)
+* **Invalid** : अमान्य , We can use the translation of 
+"incorrect" here, but this translation means "unacceptable", which is better as 
+we are not able to accept the input, but it need not mean the input is wrong , 
+बॉट का नाम अमान्य है (Bot name is invalid) 
+* **Customization** : अनुकूलन , Possibly the only translation 
+available. Trying to translate "customize" also returns a form of this word 
+itself. , अपने अनुकूलन देखें (See your 
+customization)
+* **I want** : मुझे ```something``` चाहिए , Just 
+like Marathi and other Indian languages, the sentence formation follows the 
+pattern: ```subject object action```. Therefore if there is an object, it should 
+come between मुझे and चाहिए , मैं एक मेल 
+भेजना चाहता हूं (I want to send a mail)
+* **User** : 
+उपयोगकर्ता , Like subscribe, the translation of user is from 
+the root word उपयोग , which means use. Therefore 
+उपयोगकर्ता means the one who uses , 
+उपयोगकर्ता को खोजें (Search user) 
+* **Person/People** : व्यक्ति / लोग , Like Marathi, the 
+singular and plural forms are totally unrelated to each other. Also, they are 
+the most common/formal translations that can be used , इस विषय 
+पर चर्चा करने वाले लोग देखें (See 
+people discussing this topic) 
+## Other (अन्य)
+* **You** : आप , Note 
+that you can be also translated as तुम, but that is used in informal 
+contexts. No other words are available for normal use besides those two. , 
+हम आपको एक मेल भेज देंगे (We will send you 
+a mail)
+* **We** : हम , Only translation of we, the other translations are 
+just the root word and the suffixes added so that it means "we all" or "we 
+people", which is unneccassary in Zulip , हम जल्द ही 
+आपके खाते को जोड़ देंगे (We will add your 
+account soon) 
```

Comment:
Okay, so two things that are wrong here:
* Replace `unneccassary` with `unnecessary`
* हम जल्द ही आपके खाते को जोड़ देंगे is wrong. जोड़ means join, not add. A better word here would be शामिल. :)

Reply:
Changed both of them